handlers/bulk_refunds.go

```diff
@@ -124,3 +125,26 @@ func closeFile(file multipart.File) {
 		log.Error(fmt.Errorf("error closing file: %w", err))
 	}
 }
+
+// HandleGetRefundStatuses retreives payments that are pending refund
+func HandleGetRefundStatuses(w http.ResponseWriter, req *http.Request) {
```

Comment:
We should add unit tests for this method because Sonar has failed the quality checks due to missing tests